src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
They are not siblings but a parent and its child
```suggestion
<html lang="en">
  <head>
```